### ISSUE TITLE
[Bug]: system_settings is not set in settings_store after installation

### ISSUE THREAD

**Issue by HemiCZ** [pimcore/pimcore#15596]
### Pimcore version

11.0.5

### Steps to reproduce

```
system_settings:
    write_target:
        type: 'settings-store'
    read_target:
        type: 'settings-store'
```

`./vendor/bin/pimcore-install`

### Actual Behavior

Warning: Undefined array key "general"

### Expected Behavior

Add system_settings to settings_store after installation.

**Comment by cadeyrn** [comment]
I have the same issue after upgrading my test instance from Pimcore 10.6.6 to 11.0.5.

**Comment by ayoob-lfc** [comment]
I had the same issue. A quick fix would be to install everything as normal.

Then make the following changes:

```
        system_settings:
                write_target:
                    type: 'settings-store'
```

In your admin go to "System Settings" and click "Save".

(Once saved, an entry will be created in settings_store)

Go back to config and update the setting to:

```
        system_settings:
                write_target:
                    type: 'settings-store'
                read_target:
                    type: 'settings-store'
```

**Comment by dvesh3** [comment]
Reproducible with Skeleton

**Comment by dvesh3** [comment]
Fixed by #15822
